src/services/http.hook.ts

```diff
@@ -0,0 +1,30 @@
+export const useHttp = () => {
+  const request = async (
```

Comment:
Can this hook be used with generic type? It means we will have better typing for our methods